lib/cinch/plugins/word_game.rb

```diff
@@ -128,7 +129,7 @@ def start(m)
       if @game
         m.reply "There's already a game running!"
       else
-        @game = Game.new(@dict, @ref_dict)
+        @game = Cinch::Plugins::WordGames::Game.new(@dict, @ref_dict)
```

Comment:
This will have to change given my previous comment (extend the base class)... the extended Game class probably should stay where it originally was (in the wordgame plugin) (this file)